frontend/src/app/admin.tsx

```diff
@@ -1,6 +1,11 @@
 import React, { Component } from 'react';
-import { Button } from '@patternfly/react-core';
 import { Redirect } from 'react-router-dom';
+import { Button } from '@patternfly/react-core';
+import { Grid, GridItem } from '@patternfly/react-core';
+import browseri from './sample.svg';
```

Comment:
You are importing these svgs (console1.svg, etc) but they are not being used, instead it looks like the SVG is copied on the admin.tsx as well. I would delete the paths with the svg vectors from this file and use the imported ones.

Reply:
Hey Joseph!
I have addressed the issue.
The images are appearing now.